gluefactory/models/extractors/joint_point_line_extractor.py

```diff
@@ -197,14 +194,23 @@ def _init(self, conf):
         if conf.checkpoint is not None and Path(conf.checkpoint).exists():
             logger.warning(f"Load model parameters from checkpoint {conf.checkpoint}")
             chkpt = torch.load(conf.checkpoint, map_location=torch.device("cpu"))
+
+            # remove mlp weights from line detection
+            chkpt["model"] = {k: v for k, v in chkpt["model"].items() if not ("mlp" in k)}
+
             self.load_state_dict(
-                chkpt["model"], strict=False
+                chkpt["model"], strict=True
             )  # set to false as otherwise error raised for missing mlp weights
```

Comment:
if you change to strict=True, please adapt the comment, to why strict is used

Reply:
done